foyer/atomtyper.py

```diff
@@ -48,8 +48,8 @@ def find_atomtypes(structure, forcefield, max_iter=10):
                 system_elements.add(name)
         else:
             atomic_number = atom_data.atomic_number
-            if 0 < atomic_number <= pt.KNOWN_ELEMENTS:
-                element = pt.Element[atomic_number]
+            if 0 < atomic_number <= len(ele.Elements[0]):
```

Comment:
Why not something like:

```python
try:
    element = ele.element_from_atomic_number(atomic_number).symbol
except ElementError:
    raise FoyerError(...)
```
```

Reply:
@daico007 are you planning on changing this? Your test right now assumes there are no "skipped" elements in the list from `ele`. Ele also provides the `ElementError` for exactly this purpose.

Reply:
Sorry, I missed this comment. I will address this in the next commit.